php/class-fieldmanager-select.php

```diff
@@ -7,48 +12,53 @@
  * options (values) via an array or via a dynamic
  * {@link Fieldmanager_Datasource}, like {@link Fieldmanager_Datasource_Post},
  * {@link Fieldmanager_Datasource_Term}, or {@link Fieldmanager_Datasource_User}.
- *
- * @package Fieldmanager_Field
  */
 class Fieldmanager_Select extends Fieldmanager_Options {
 
 	/**
+	 * Override $field_class.
+	 *
 	 * @var string
-	 * Override $field_class
 	 */
 	public $field_class = 'select';
 
 	/**
+	 * Should we support type-ahead? i.e. use chosen.js or not.
+	 *
 	 * @var boolean
-	 * Should we support type-ahead? i.e. use chosen.js or not
 	 */
-	public $type_ahead = False;
+	public $type_ahead = false;
 
 	/**
+	 * Send an empty element first.
+	 *
 	 * @var boolean
-	 * Send an empty element first
 	 */
-	public $first_empty = False;
+	public $first_empty = false;
 
 	/**
+	 * Tell FM to save multiple values.
+	 *
 	 * @var boolean
-	 * Tell FM to save multiple values
 	 */
 	public $multiple = false;
 
 	/**
-	 * Override constructor to add chosen.js maybe
-	 * @param string $label
-	 * @param array $options
+	 * Override constructor to add chosen.js maybe.
+	 *
+	 * @param string $label   The form label.
+	 * @param array  $options The form options.
 	 */
 	public function __construct( $label = '', $options = array() ) {
 
 		$this->attributes = array(
-			'size' => '1'
+			'size' => '1',
 		);
 
-		// Add the Fieldmanager Select javascript library
-		fm_add_script( 'fm_select_js', 'js/fieldmanager-select.js', array(), '1.0.2', false, 'fm_select', array( 'nonce' => wp_create_nonce( 'fm_search_terms_nonce' ) ) );
+		// Add the Fieldmanager Select javascript library.
```

Comment:
We use "Javascript" most other places in the PR. (I would also think, given the spelling on the [JS docs page](https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/javascript/), that core also prefers "JavaScript").